### ISSUE TITLE
New feature: Reorder tabs in session

### ISSUE THREAD

**Issue by iago-costa** [zellij-org/zellij#3080]
### Discussed in https://github.com/zellij-org/zellij/discussions/3079

<div type='discussions-op-text'>

<sup>Originally posted by **iago-costa** January 22, 2024</sup>
## Feature: Reorder tabs

1. Using drag and drop
2. By alphabet
3. By set index in CLI

![image](https://github.com/zellij-org/zellij/assets/43588158/c6fd7e43-365e-4ef6-98e2-d906cdeac8db)
</div>

**Comment by janos-r** [comment]
This is already being discussed for a long time in
https://github.com/zellij-org/zellij/issues/1656
and https://github.com/zellij-org/zellij/issues/1654

And recently finally made a PR in
https://github.com/zellij-org/zellij/pull/3047

You should look for similar issues before posting a new empty discussion and new redundant issue. Did you really think in all the years nobody else thought of moving tabs? It is just for some reason a difficult feature.

**Comment by Friedrich42** [comment]
I guess this can be closed

**Comment by imsnif** [comment]
Hey, just happened upon this thread and wanted to chime in:

@iago-costa - please don't feel bad about opening a new issue for this. I'm happy you enjoy Zellij and like it enough to want to let the developers know about the features you'd like to see.

Otherwise - while I appreciate everyone's willingness to want to consolidate issues so that discussions do not fragment - let's please try to create a kind and welcoming environment around this project. To me, this is far more important than any issue duplication. Thanks.

**Comment by Friedrich42** [comment]
@imsnif
I love you so much, dude C:

I wanted to say that the previous message was a little bit threatening, but thanks for writing this.

**Comment by matthiasbeyer** [comment]
I came here to request this as well and found that #3047 was already merged (although not released yet).

Time to close this issue? Not sure what your policy here is, whether to close issues once they are implemented or only after the features are released... 

I am really looking forward to this feature! :heart:

**Comment by Victor-N-Suadicani** [comment]
For anyone else who lands here when trying to figure out how to do this: ALT + I (move left) and ALT + O (move right) is the shortcut (maybe it'd be nice if this was hinted at in the bottom bar in the tab or move menu?).
